doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_mongo_sink.md

```diff
@@ -324,6 +324,8 @@ When datamodel changes Cygnus tries to recreate index (delete current and create
 | mongo\_ssl\_invalid\_host\_allowed | no | false | Allow invalid host name in mongo SSL connections |
 | mongo\_ssl\_keystore\_path\_file | no | <i>empty</i> | Java SSL KeyStore path file (JKS file). A JKS file could be create from a certificate file using keytool: ```keytool -importkeystore -srckeystore certificate.p12 -srcstoretype pkcs12 -destkeystore mongo_ssl_keystore.jks```  |
 | mongo\_ssl\_keystore\_password | no | <i>empty</i> | Java SSL KeyStore password for keystore file (JKS file).  |
+| mongo\_ssl\_truststore\_path\_file | no | <i>empty</i> | Java SSL TrustStore for CAs path file (JKS file).  |
+| mongo\_ssl\_truststore\_password | no | <i>empty</i> | Java SSL TrustStore password for keystore file (JKS file).  |
```

Comment:
Should be added to CNR entry?

Reply:
Done in https://github.com/telefonicaid/fiware-cygnus/pull/2388/commits/3356903070c235af95d9905ab09631cb829c44fa